apps/src/tutorialExplorer/tutorialSet.jsx

```diff
@@ -62,14 +64,23 @@ export default class TutorialSet extends React.Component {
           changeTutorial={this.changeTutorial}
           grade={this.props.grade}
         />
-        {this.props.tutorials.map(item => (
-          <Tutorial
-            item={item}
-            filters={this.props.filters}
-            key={item.code}
-            tutorialClicked={this.tutorialClicked.bind(this, item)}
-          />
-        ))}
+        {this.props.tutorials.map(item =>
+          this.props.specificLocale ? (
+            <TutorialSpecificLocale
+              item={item}
+              filters={this.props.filters}
```

Comment:
`TutorialSpecificLocale` doesn't take the `filters` prop.

Reply:
Or, if you wanted these to be interoperable, maybe we should use them as such...

```jsx
const TutorialComponent = this.props.specificLocale ? TutorialSpecificLocale : Tutorial;
//...
{this.props.tutorials.map(item => (
  <TutorialComponent
    // ...
  />
))}
```
...and DRY the props-passing.

Reply:
Great idea, will do!